recipe/meta.yaml

```
@@ -31,15 +31,14 @@ requirements:
 
 test:
   imports:
-    - scimma
-    - scimma.client
-    - scimma.client.publish
+    - hop
+    - hop.publish
 
```

I missed this when I added `scimma subscribe`, so we should include that test here:
```suggestion
    - hop.subscribe

```